examples/benchmark.jl

```diff
@@ -25,26 +29,32 @@ function two_solvers()
   bmark_and_profile(solvers,
                     (MathProgNLPModel(eval(p)(n), name=string(p)) for p in mpb_probs),
                     bmark_args=bmark_args, profile_args=profile_args)
+  png("two-solvers")
 end
 
 # Example 2: benchmark one solver on problems written in two modeling languages
 function two_languages()
-  probs = ampl_probs ∩ mpb_probs
-  stats = Dict{Symbol, Array{Int,2}}()
+  stats = Dict{Symbol, DataFrame}()
   stats[:trunk_ampl] = solve_problems(trunk,
                                       (AmplModel(p) for p in ampl_probs),
                                       skipif=model -> !unconstrained(model))
   stats[:trunk_mpb] = solve_problems(trunk,
                                      (MathProgNLPModel(eval(p)(n), name=string(p)) for p in mpb_probs),
                                      skipif=model -> !unconstrained(model))
   profile_solvers(stats, title="f+g+hprod")
+  png("two-languages")
 end
 
 # Example 3: benchmark one solver with different options on a set of problems
 function solver_options()
-  stats = Dict{Symbol, Array{Int,2}}()
+  stats = Dict{Symbol, DataFrame}()
   probs = (MathProgNLPModel(eval(p)(n), name=string(p)) for p in mpb_probs)
-  stats[:trunk] = solve_problems(trunk, probs, skipif=model -> !unconstrained(model), nm_itmax=5)
-  stats[:trunk_monotone] = solve_problems(trunk, skipif=model -> !unconstrained(model), monotone=true)
+  stats[:trunk_monotone] = solve_problems(trunk, probs, skipif=model -> !unconstrained(model), nm_itmax=5)
+  stats[:trunk_nonmonotone] = solve_problems(trunk, probs, skipif=model -> !unconstrained(model), monotone=false)
```

Comment:
What I mean is that `nm_itmax=5` should be an option in the non-monotone version.

Reply:
Fixed.